app/features/innlogget/personinformasjon/schema.ts

```diff
@@ -22,8 +22,10 @@ const BarnRelasjonSchema = z.object({
  * Schemaet for responsen fra personinformasjon API-et.
  */
 export const PersoninformasjonSchema = z.object({
-  påloggetPerson: PersonSchema,
-  barnRelasjon: z.array(BarnRelasjonSchema),
+  påloggetBruker: PersonSchema.and(
+    z.object({ inntekt: z.number().int().nonnegative().nullable() })
+  ),
+  relasjoner: z.array(RelasjonSchema),
```

Comment:
Det er vel mulig at det kommer en del andre felter her. Hva med å gruppere personinfo videre? (kanskje denne kommentaren egentlig hører hjemme i api-repo, men skriver her likevel :))
```suggestion
  påloggetBruker: z.object({
    person: PersonSchema,
    inntekt: z.number().int().nonnegative().nullable(),
  }),
  relasjoner: z.array(RelasjonSchema),
```